tests/data/REFERENCE_PPP_o2_glc_EcoliCore.csv

```diff
@@ -1,401 +1,401 @@
 ,EX_o2_LPAREN_e_RPAREN_,EX_glc_LPAREN_e_RPAREN_,objective_lower_bound,objective_upper_bound
-0,-60.00000000000003,-10.0,0.0,-1.0867823178038478e-15
```

Comment:
@phantomas1234 the reference 2d-ppp has listed objective upper bound as negative (optimizing for biomass) which was a mistake? or am I misunderstanding something? I could not reproduce this result so have updated with new reference, please confirm (or suggest correction)